### ISSUE TITLE
Feature Request: Firefox tabs

### ISSUE THREAD

**Issue by jdamboeck** [G-dH/windows-search-provider#4]
It would be most awesome if this extension could find and target open firefox tabs. I know it is probably out of scope. Just threwing it out there.

**Comment by G-dH** [comment]
It's a task that involves more than just a GNOME Shell extension; it also requires an extension in Firefox and a connector in the system.
https://github.com/harshadgavali/searchprovider-for-browser-tabs/

I'm using a Firefox extension that lets me add a string prefix to its window titles. This feature enables me to organize tabs into groups, which I can then navigate between using window search. Not perfect, but better than nothing.

**Comment by jdamboeck** [comment]
Yeah. I used that before.. but the extension does not work anymore... Not updated for a while.
